Server/src/main/java/app/ModelController.java

```diff
@@ -248,4 +261,16 @@ public Instances createInstancesFromAssets(List<Asset> assets) {
         }
         return data;
     }
+
+    public void modelToEvaluate(TrainedModel trainedModel) throws Exception {
```

Comment:
Add comments

Reply:
@t98b There's multiple review by Jeff asking for comments. Did you add those? Look at the comments and resolve the conversations.